GitVersion.yml

```diff
@@ -0,0 +1,40 @@
+mode: Mainline
+assembly-versioning-scheme: MajorMinorPatch
+tag-prefix: release\/
+ignore:
+  sha: []
+
+branches:
+  master:
+    regex: ^master$|^main$
+    tag: ''
+    increment: Patch
+    prevent-increment-of-merged-branch-version: true
+    track-merge-target: false
+    tracks-release-branches: false
+    is-release-branch: true
+  pull-request:
+    regex: ^(pull|pull\-requests|pr)[/-]
+    tag: pr
+    increment: Inherit
+    prevent-increment-of-merged-branch-version: false
+    tag-number-pattern: '[/-](?<number>\d+)[-/]'
+    track-merge-target: false
+    tracks-release-branches: false
+    is-release-branch: false
+  hotfix:
+    regex: ^hotfix(es)?[/-]
+    tag: beta
+    increment: Patch
+    prevent-increment-of-merged-branch-version: false
+    track-merge-target: false
+    tracks-release-branches: false
+    is-release-branch: false
+  feature:
+    regex: ^(personal|dev|feature|auto\-nuget\-update)[/-]
```

Comment:
This is currently only used by the build pipeline, your fork should still use the pr config when submitting pull requests. I don't see a lot of use for this, but it might apply if you wanted to create build assets for features. As long as you're building with our microsofthealthoss devops it should be ok.